spec/models/npq_validation_data_spec.rb

```diff
@@ -11,4 +11,26 @@
       yes_over_two_years: "yes_over_two_years",
     ).backed_by_column_of_type(:text)
   }
+
+  describe "callbacks" do
+    subject { create(:npq_validation_data) }
+
+    context "on creation" do
+      it "fires NPQ::StreamBigQueryEnrollmentJob" do
+        expect {
+          subject
+        }.to change(Delayed::Job, :count).by(1)
+      end
+    end
+
+    context "when lead_provider_approval_status is modified" do
+      it "fires NPQ::StreamBigQueryEnrollmentJob" do
+        subject
+
+        expect {
+          subject.update(lead_provider_approval_status: "accepted")
+        }.to change(Delayed::Job, :count).by(1)
+      end
+    end
+  end
```

Comment:
Can we have a test showing when the BigQuery callback is not fired?

Reply:
I've now added a test for this